### DISCUSSION TITLE
The most powerful coordinator.

### DISCUSSION THREAD

**Discussion by Jaguaza** [Koenkk/zigbee2mqtt#8891] [Devices]
Good Morning. 
 I need help, I must change my cc2531, which coordinator do you recommend.

 which one is the most powerful right now?

**Comment by Koenkk** [comment]
See https://www.zigbee2mqtt.io/getting_started/what_do_i_need.html#supported-zigbee-adapter (basically all CC2652/CC1352 based coordinators)

**Reply by Jaguaza** [reply]
> See https://www.zigbee2mqtt.io/getting_started/what_do_i_need.html#supported-zigbee-adapter (basically all CC2652/CC1352 based coordinators)

Thanks!!!!!

**Reply by Jaguaza** [reply]
> See https://www.zigbee2mqtt.io/getting_started/what_do_i_need.html#supported-zigbee-adapter (basically all CC2652/CC1352 based coordinators)

This would be nice?, they showed it to me today in a group.

https://sonoff.tech/product/diy-smart-switch/sonoff-dongle-plus/

Thanks @Koenkk.

**Reply by Koenkk** [reply]
It looks good, haven't tested it myself but ITEAD has send me one for evaluation. If it works as expected it will add it to the supported adapters list

**Reply by Jaguaza** [reply]
About the limitation of 20 children that can be unlocked to support more than 100 children, you know something?

**Reply by Hedda** [reply]
See "Z-Stack_3.x.0" for CC2652 https://github.com/Koenkk/Z-Stack-firmware/blob/master/coordinator/README.md

**Reply by Jaguaza** [reply]
@Hedda, @Koenkk this firmware could be put in to unlock it.  I bought it this morning and I would like to have everything clear.

**Reply by Hedda** [reply]
Yes, if Sonoff is using the same pin standard as the other dongles then using CC1352P2_CC2652P_launchpad_*.zip from https://github.com/Koenkk/Z-Stack-firmware/tree/master/coordinator/Z-Stack_3.x.0/bin should work since it has a CC2652P chip

Itead even posted a link to their own flashing guide which refer to suing firmware hosted by Koenkk so I assume they use same standard:

https://sonoff.tech/product-review/zigbee-3-0-usb-dongle-plus/

https://sonoff.tech/wp-content/uploads/2021/09/Zigbee-3.0-USB-dongle-plus-firmware-flashing-1-1.docx

PS: You can also post feedback here once you tested it if you get it first -> https://github.com/Koenkk/Z-Stack-firmware/pull/313

**Reply by Koenkk** [reply]
ITEAD confirmed to me that `CC1352P2_CC2652P_launchpad_coordinator_20210120` is flashed on it out-of-the-box

**Reply by Jaguaza** [reply]
But I imagine that it will be limited to 20 children as sonoff says on its website.

**Reply by Koenkk** [reply]
I'm not sure why they say that, `CC1352P2_CC2652P_launchpad_coordinator_20210120` doesn't have such limit. I will ask them

**Reply by Jaguaza** [reply]
Thank you, we hope many people answer.

**Reply by Jaguaza** [reply]
@Koenkk 

![50A1ABE2-2AF1-480B-9A5E-54CCC9DC34F1](https://user-images.githubusercontent.com/46264854/135151755-a231e5b7-99df-4398-b807-be11ece38854.jpeg)

**Reply by Jaguaza** [reply]
> I'm not sure why they say that, `CC1352P2_CC2652P_launchpad_coordinator_20210120` doesn't have such limit. I will ask them

@Koenkk We know something about the limitation, did you answer already itead?  Tomorrow I get mine and I would like to know if I would have to flash, how and with what.  Thank you very much for your work.

**Reply by Koenkk** [reply]
I got an answer for them, the limitations listed on their site are wrong. It has the same limitations as any other device running zstack 3.x.0, limits can be found in this table: https://github.com/Koenkk/Z-Stack-firmware/tree/master/coordinator

**Reply by Jaguaza** [reply]
> I got an answer for them, the limitations listed on their site are wrong. It has the same limitations as any other device running zstack 3.x.0, limits can be found in this table: https://github.com/Koenkk/Z-Stack-firmware/tree/master/coordinator

Thanks!!!!.

**Comment by Hedda** [comment]
@Jaguaza FYI, more discussion + much info collected on that specific dongle here -> https://github.com/Koenkk/zigbee2mqtt/discussions

**Reply by Jaguaza** [reply]
Thank you, I will follow you, I already knew that information from the sonoff website.  I hope that @koenkk will answer before me, because I wrote to support for the limitation and I have not had an answer yet.  @Koenkk you had more luck?

**Reply by Koenkk** [reply]
I believe they are confused with the CC2531 limitation. This is what I got back (send a reply but they are on holiday till 7 october).

<img width="942" alt="Screenshot 2021-09-30 at 19 39 52" src="https://user-images.githubusercontent.com/2892853/135504318-52ab85f5-25e3-4e6d-97b3-0c2a9118e5e4.png">

The correct limits are shown here: https://github.com/Koenkk/Z-Stack-firmware/tree/master/coordinator (since they are using my fws)

**Reply by Hedda** [reply]
Yeah as far as I can tell https://www.home-assistant.io/integrations/zha only mentions CC2531 limits as an example:

https://www.home-assistant.io/integrations/zha#using-router-devices
